pkg/config/system_probe.go

```diff
@@ -149,9 +149,9 @@ func InitSystemProbeConfig(cfg Config) {
 	cfg.BindEnv(join(netNS, "enable_http_monitoring"), "DD_SYSTEM_PROBE_NETWORK_ENABLE_HTTP_MONITORING")
 	cfg.BindEnv(join(netNS, "enable_https_monitoring"), "DD_SYSTEM_PROBE_NETWORK_ENABLE_HTTPS_MONITORING")
 
-	cfg.BindEnvAndSetDefault(join(spNS, "enable_go_tls_support"), false)
+	cfg.BindEnvAndSetDefault(join(smNS, "enable_go_tls_support"), false)
```

Comment:
not related to the PR, please undo

Reply:
relevant to any not necessary change for gotls

Reply:
I move the changed to another PR that need to be merged after jattach